### PR TITLE
PISCIS : fishpeople - new donator only roundstart race!

### PR DESCRIPTION
**Pull request by Djiq**
What are their features - their features are lizard but opposite meaning they have slight resistance to cold damage but a weakness to heat, meaning they are well suited to job that often puts them in cold temperatures - look at chef who has to go to freezer often for example. Due to their simple neural system they are also stunned for 10% more. They also require a constant intake of H2O to deliver oxygen to their brain, if H2O is less than 16 kpa then they start to choke. They can survive some time without H2O but it is recommended to get em back on it as soon as possible. Even tough they require H2O to breathe they also require oxygen. Same as any human, they just dont tolerate atmospheres without water.

I did it for money yet again.

here is the thread :

https://forums.yogstation.net/threads/35-fishpeople-a-donator-race-ss13-server.22396/

https://i.gyazo.com/116f3c2effea62a309905608e807ad5b.mp4

https://i.gyazo.com/9055d9c25edabd11335eefe0521c6fa1.mp4

https://i.gyazo.com/40fccf9964d01876e477fc5c2aa595f6.mp4

image

image

image

image

#### Changelog

:cl:  
rscadd: NEW DONATOR ONLY RACE : piscis - fresh from the sea water!
rscadd: added a system for adding donator only races and tied it to policy while i was at it.
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
but what do they even look like

**Comment by jcatjmeow** [comment]
Do not personally support donor-only races.

Will check the code for quality later. In the meantime, do you have any screenshots of the mechanics in action?

**Comment by Djiq** [comment]
lemme pull up the sprites gimme a second

File: code/modules/language/piscis.dm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/datum/language/deepwater
+	name = "Deep Tongue"
+	desc = "The common language of deep water creatures, consisting of periodic bubbling."
+	speech_verb = "bubbles"
+	ask_verb = "sonars"
+	exclaim_verb = "bobbles"
+	key = "f"
+	flags = TONGUELESS_SPEECH
```
Comment (jcatjmeow) [inline code/modules/language/piscis.dm:8]:
```suggestion
	flags = TONGUELESS_SPEECH
```
Why?

**Comment by Djiq** [comment]
and regarding the bounty - they got approved thats all i really care about 
money is money man

**Comment by jcatjmeow** [comment]
The label is because it's wip not because it will never be merged

**Comment by Marmio64** [comment]
why do they look like creatures from the black lagoon

**Comment by Xoxeyos** [comment]
They look like shit, I doubt this will fulfill the bounty, make them look better, and not like one mess of colors, atleast I can see a tail and a mouth. I suggest you take a look here, if you want more help with your sprites. https://discord.gg/jYquSmy (Spriter's Bay)

**Comment by patpol4** [comment]
These sprites are horrible

**Comment by Partheo** [comment]
Can you post some images of a few different colors besides this light blue

**Comment by UselessTheremin** [comment]
Yeah I don’t want to be harsh but this is nothing like what the bounty wanted, it was supposed to be more of a plasma man clone, but this is just a lizard clone and it is very boring.

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

File: code/modules/language/piscis.dm (hunk 1 above)
Comment (Hopekz) [reply code/modules/language/piscis.dm:8]: they're fish

**Comment by Djiq** [comment]
> Yeah I don’t want to be harsh but this is nothing like what the bounty wanted, it was supposed to be more of a plasma man clone, but this is just a lizard clone and it is very boring.

image

**Comment by neeshacark** [comment]
> > Yeah I don’t want to be harsh but this is nothing like what the bounty wanted, it was supposed to be more of a plasma man clone, but this is just a lizard clone and it is very boring.
> 
> image

someone poke xantam. water vapor needs a use and I don't see why NEGATIVES for a donor race bad

**Comment by Hopekz** [comment]
Fish people were a terrible idea from the start but this PR isn't bad.

The sprites aren't too great but not everyone is a spriter + coder so I'm pretty sure someone will care about remaking their visuals sometime like with other early mutant races.

**Comment by Hopekz** [comment]
Also, don't let the down doinks discourage you this whole idea wasn't too popular from the start from what I can tell
image

**Comment by jcatjmeow** [comment]
Can you please provide a clip of gameplay with the new race? Either a gif/video or a series of screenshots, showing the new alerts, mechanics, sprites, etc.
@EdgeLordExe

**Comment by Djiq** [comment]
> Can you please provide a clip of gameplay with the new race? Either a gif/video or a series of screenshots, showing the new alerts, mechanics, sprites, etc.
> @EdgeLordExe

im currently trying to debug the tails of them disappearing when changing clothes may take some time since the code is grade A shitcode

**Comment by swissloaf** [comment]
> > > Yeah I don’t want to be harsh but this is nothing like what the bounty wanted, it was supposed to be more of a plasma man clone, but this is just a lizard clone and it is very boring.
> > 
> > 
> > image
> 
> someone poke xantam. water vapor needs a use and I don't see why NEGATIVES for a donor race bad

Don't listen to him, we agreed in staff channels that it could be a plasmaman resprite with water vapor tanks if that's possible for you to do

**Comment by Mqiib** [comment]
image

Shouldnt the bit at the very end there be roundstart_races and not donator_races?

**Comment by Djiq** [comment]
> image
> 
> Shouldnt the bit at the very end there be roundstart_races and not donator_races?

yeah

**Comment by Djiq** [comment]
@jcatjmeow i updated the pr body to feature gifs and pictures of piscis

**Comment by Marmio64** [comment]
they look MUCH better now at least

**Comment by swissloaf** [comment]
give them fish bowl head

**Comment by Djiq** [comment]
dont feel like solving the conflicts rn, will reopen once i do

**Pull request closed without merging**